packages/astro/src/assets/utils/svg.ts

```diff
@@ -0,0 +1,54 @@
+import { parse, renderSync } from 'ultrahtml';
```

Comment:
Just a callout that I maintain `ultrahtml`—this could easily be swapped for another XML parser if preferred, but keep in mind that it will be bundled into the server runtime so we should try to keep a light footprint.